### PR TITLE
Checker Option page landmarks/regions design and implementation 

### PR DESCRIPTION
**Pull request by zhenwan**
Issue: https://github.ibm.com/IBMa/E2E/issues/1370
landmark/region design: https://app.mural.co/t/ibmaccessibility7766/m/ibmaccessibility7766/1591812623557/3372bde26c50f3afbbc6845bad969c7c45c5bf04

### CODE REVIEW

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx [hunk 1]
```diff
@@ -114,7 +116,7 @@ class OptionsApp extends React.Component<{}, OptionsAppState> {
       selected_archive: item.selectedItem,
       rulesets,
       selected_ruleset,
-      show_notif: false
+      show_notif: false,
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/options/OptionsApp.tsx:119]: Any reason a comma was added here?

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx [hunk 2]
```diff
@@ -150,115 +152,136 @@ class OptionsApp extends React.Component<{}, OptionsAppState> {
...
-                    }}
-                  />
-                </div>
-              ) : (
-                ""
-              )}
-              <div className="buttonRow">
-                <Button
+                <Dropdown
+                  ariaLabel="Deployment selection dropdown"
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/options/OptionsApp.tsx:190]: Why is there both a label and an ariaLabel? Also, you should not put 'dropdown' in the label, much as you wouldn't add 'dropdown' to the visible label. Widget type is conveyed to the screen reader through the widget role.

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx [hunk 3]
```diff
@@ -150,115 +152,136 @@ class OptionsApp extends React.Component<{}, OptionsAppState> {
...
+                  .
+                </p>
+                <h2>Supported rule sets</h2>
+                <p>
+                  Choose which rule set to use. This will affect the issues
+                  detected.
+                </p>
+
+                <Dropdown
+                  ariaLabel="ruleset selection dropdown"
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/options/OptionsApp.tsx:221]: Why is there both a label and an ariaLabel? Also, you should not put 'dropdown' in the label, much as you wouldn't add 'dropdown' to the visible label. Widget type is conveyed to the screen reader through the widget role.

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx (hunk 1 above)
Comment (zhenwan) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx:119]: Strange, VS Code 'Format Document' added that it.
Removed

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx (hunk 2 above)
Comment (zhenwan) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx:190]: For Type script, it looks label and ariaLabel are both required by Dropdown component ,  removing either one of them will cause compilation issue, even the Carbon's doc saying ariaLabel is optional,
https://react.carbondesignsystem.com/?path=/story/dropdown--default

I know the voice over speaks both arialLabel and label. it is kind of redundant. To work around, I will keep the arialLabel, but assign an empty string to it like this: ariaLabel="". Hope it is OK.

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx (hunk 3 above)
Comment (zhenwan) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx:221]: For Type script, it looks label and ariaLabel are both required by Dropdown component ,  removing either one of them will cause compilation issue, even the Carbon's doc saying ariaLabel is optional,
https://react.carbondesignsystem.com/?path=/story/dropdown--default

I know the voice over speaks both arialLabel and label. it is kind of redundant. To work around, I will keep the arialLabel, but assign an empty string to it like this: ariaLabel="". Hope it is OK.

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx (hunk 2 above)
Comment (cpandhi) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx]: @zhenwan just fyi, aria-label overrides the native label, assigning ariaLabel="" is not a good idea.

Comment (cpandhi) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx]: also, the attribute is aria-label and not ariaLabel

Comment (zhenwan) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx]: Using ariaLabel="" it to work around Typescript issue. please see my comment above.

"ariaLabel" is a property of Dropdown component. We can not use ' aria-label'

File: accessibility-checker-extension/src/ts/options/OptionsApp.tsx (hunk 3 above)
Comment (tombrunet) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx]: I took a look at the storybook. Looks like they set an aria-label on the div surrounding the button with no role. Looks like bad implementation. Try `ariaLabel={null}`.

Comment (zhenwan) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx]: tried ariaLabel={null}, it is not taken: 
image

So I changed to ariaLabel={undefined}, and it looks fine.

Comment (tombrunet) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx]: Did anyone have a chance to check it with a screen reader? The DOM looked okay to me, but would be nice to confirm that there isn't something weird going on.

Comment (zhenwan) [reply accessibility-checker-extension/src/ts/options/OptionsApp.tsx]: I did Voiceover checking ( have not got time to install JAWS), the label read correctly.

**Pull request merged**